charlib/characterizer/port.py

```
@@ -70,6 +70,13 @@ def __init__(self, name: str, direction: str, role: str, trigger: bool):
     def __repr__(self) -> str:
         return f'Port({self.name}, {self.direction}, {self.role}, {self.trigger})'
 
+    def __eq__(self, other: Port) -> bool:
```

Ruff (0.15.17)

[error] Undefined name `Port`

(F821)